packages/dart/npt_flutter/lib/features/tray_manager/widgets/tray_manager.dart

```diff
@@ -16,13 +19,62 @@ class TrayManager extends StatefulWidget {
 
 class _TrayManagerState extends State<TrayManager>
     with TrayListener, WindowListener {
+  /// Must strongly type [context] here or Dart will infer the wrong type for
+  /// the [.read()] extension which causes an error
+  void reloadTray(BuildContext context, _) {
+    context.read<TrayCubit>().reload();
+  }
+
```

Comment:
This is the listener which triggers tray reloads whenever a piece of state changes.